lib/cocoapods/generator/embed_frameworks_script.rb

```diff
@@ -45,6 +45,15 @@ def script
 
           install_framework()
           {
+            if [ -e "$1" ]; then
+              version=$(defaults read "${PODS_ROOT}/Target Support Files/$(basename -s .framework "$1")/Info.plist" CFBundleShortVersionString)
```

Comment:
I think we might be entering a bigger discussion here, what does this PR exactly try to resolve? In CocoaPods 1.3.0 we added input and output paths to the xcode phase and this script will be skipped from running unless the contents have changed.

Are you trying to achieve a performance improvement? If so have you measured the impact of your change?